src/woo_publications/api/drf_spectacular/extensions.py

```diff
@@ -0,0 +1,20 @@
+from drf_spectacular.contrib.django_filters import DjangoFilterExtension
+
+from woo_publications.api.utils import underscore_to_camel
+
+
+class CamelizeFilterExtension(DjangoFilterExtension):
+    priority = 1
+
+    def get_schema_operation_parameters(self, auto_schema, *args, **kwargs):
+        """
+        camelize query parameters
+        """
+        parameters = super().get_schema_operation_parameters(  # pragma: no cover
+            auto_schema, *args, **kwargs
+        )
+
+        for parameter in parameters:  # pragma: no cover
+            parameter["name"] = underscore_to_camel(parameter["name"])
+
+        return parameters  # pragma: no cover
```

Comment:
```suggestion
    def get_schema_operation_parameters(
        self, auto_schema, *args, **kwargs
    ):  # pragma: no cover
        """
        camelize query parameters
        """
        parameters = super().get_schema_operation_parameters(
            auto_schema, *args, **kwargs
        )

        for parameter in parameters:
            parameter["name"] = underscore_to_camel(parameter["name"])

        return parameters
```
can you try this? this should exclude the entire method from coverage

note that you can run coverage checks locally too:

```bash
coverage run src/manage.py test src
coverage html
# open htmlcov/index.html in the browser
```